### ISSUE TITLE
Show tabs only from the current window

### ISSUE THREAD

**Issue by zim0369** [dvdvdmt/popup-tab-switcher#58]
I'm not sure if it's a problem due to my setup(linux+xmonad) but I'm unable to take advantage of all of the tabs from multiple windows listed altogether in my popup-tab-switcher list as the workspace doesn't switch to the one that has the window with the tab I tried to switched to. This does certainly work for windows in the same workspace but does it also work for windows in other workspaces in OSes like Windows or Mac?

Anyways, another problem is that remapping ctrl+j and ctrl+k worked for me but the list isn't persistent even while I'm holding down the ctrl key. alt+j & alt+k work absolutely fine but it shows me list of tabs from all windows thus increasing mental overhead.

TLDR; Ctrl instead of Alt on linux only half works. Show tabs only from the current window.

**Comment by Rohitth007** [comment]
Ideally, the tab switcher should only switch between the tabs in the current chrome(brave) window. If I understand correctly, that is what @zim0369 was also talking about above. The current behavior shows all tabs. Can this be fixed? @dvdvdmt

**Comment by Dozorengel** [comment]
Yep, I've got youtube music installed as PWA. And each time I focus on youtube music window, popup switches me over browser and youtube music, as it is on the tab list. It's really annoying, as a workaround I installed it in the different browser

P.S. Tabs from the private window are in the list as well, I guess, they're shouldn't be here either
